### PR TITLE
Adding the month range picker doc

### PR DESCRIPTION
**Pull request by zDudaHang**
depends on #663

### CODE REVIEW

**Comment by zDudaHang** [comment]
Não estou conseguindo resolver o seguinte problema no site:

Quando clico para olhar os componentes, o nome do Month Range Picker está saindo errado. Deveria ser Month Range Picker
Screenshot from 2021-05-03 21-30-40

Quando clico nele, fica certinho o nome:
Screenshot from 2021-05-03 21-30-46

**Comment by zDudaHang** [comment]
@gustavomoser Do I need to add the Month Range Picker in the Overview table?

**Comment by gustavomoser** [comment]
> @gustavomoser Do I need to add the Month Range Picker in the Overview table?
well, I think it would be good to keep the component with the other Pickers, but is it more intuitive?

File: site/src/pages/components/month-range-picker/index.pt.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Demo } from '../../../components/Demo'
+
+# Month Range Picker
+
+O Month Range Picker permite que o usuário selecione um período em meses através de um calendário. Por padrão, o calendário apresenta sempre o ano atual. Caso já tenha uma data inserida, o calendário apresenta o ano referente à esta data.
+
+## Best practices
+
+- O mês do valor da entrada começa com o zero, ou seja, Janeiro é representado pelo zero e Dezembro pelo 11.
```
Comment (lucasjoao) [inline site/src/pages/components/month-range-picker/index.pt.mdx:9]: For me isnt necessary to say this

**Comment by zDudaHang** [comment]
> > @gustavomoser Do I need to add the Month Range Picker in the Overview table?
> 
> well, I think it would be good to keep the component with the other Pickers, but at the same time, it's not a **Date** Picker... do you think it would be more intuitive? @zDudaHang

I think it would be more intuitive, because it is done with a separate component. This is different from what was done with the Week Range Picker, whose component is the same as the Date Range Picker.

File: site/src/pages/components/month-range-picker/index.pt.mdx (hunk 1 above)
Comment (zDudaHang) [reply site/src/pages/components/month-range-picker/index.pt.mdx:9]: I agree with you

**Comment by lucasjoao** [comment]
> @gustavomoser Do I need to add the Month Range Picker in the Overview table?

My 5 cent: put the Month Range Picker together with the date picker, everything in the end is about date and the texts are very similars. 

I will wait the definition about this to make the code review.

**Comment by gustavomoser** [comment]
> > @gustavomoser Do I need to add the Month Range Picker in the Overview table?
> 
> My 5 cent: put the Month Range Picker together with the date picker, everything in the end is about date and the texts are very similars.
> 
> I will wait the definition about this to make the code review.

makes sense, I agree

**Review by lucasjoao — APPROVED** [review]
:top:

**Review by gustavomoser — APPROVED** [review]

**Review by christianaurichzm — APPROVED** [review]

**Review by lucasjoao — APPROVED** [review]

**Pull request merged**